junit/src/main/java/org/kaazing/k3po/junit/rules/K3poRule.java

```diff
@@ -65,12 +67,14 @@
     private String scriptRoot;
     private URL controlURL;
     private SpecificationStatement statement;
+    private List<String> classOverriddenProperties;
 
     /**
      * Allocates a new K3poRule.
      */
     public K3poRule() {
         latch = new Latch();
+        classOverriddenProperties = new ArrayList<>();
```

Comment:
OK so I see there is a test case too, using the K3poRule.scriptProperty method. So ignore all these comments!